x-pack/plugins/infra/public/pages/logs/log_entry_categories/page_content.tsx

```diff
@@ -57,24 +79,52 @@ export const LogEntryCategoriesPageContent = () => {
       />
     );
   } else if (setupStatus.type === 'unknown') {
-    return <LogAnalysisSetupStatusUnknownPrompt retry={fetchJobStatus} />;
+    return (
+      <CategoriesPageTemplate PageTemplate={PageTemplate} isEmptyState={true}>
+        <LogAnalysisSetupStatusUnknownPrompt retry={fetchJobStatus} />
+      </CategoriesPageTemplate>
+    );
   } else if (isJobStatusWithResults(jobStatus['log-entry-categories-count'])) {
     return (
       <>
-        <LogEntryCategoriesResultsContent onOpenSetup={showCategoriesModuleSetup} />
+        <LogEntryCategoriesResultsContent
+          onOpenSetup={showCategoriesModuleSetup}
+          pageTitle={logCategoriesTabTitle}
+        />
         <LogAnalysisSetupFlyout allowedModules={allowedSetupModules} />
       </>
     );
   } else if (!hasLogAnalysisSetupCapabilities) {
-    return <MissingSetupPrivilegesPrompt />;
+    return (
+      <CategoriesPageTemplate PageTemplate={PageTemplate} isEmptyState={true}>
+        <MissingSetupPrivilegesPrompt />
+      </CategoriesPageTemplate>
+    );
   } else {
     return (
-      <>
+      <CategoriesPageTemplate PageTemplate={PageTemplate} isEmptyState={true}>
```

Comment:
Good spot 😬

Reply:
Interestingly I couldn't replicate the behaviour 🤔  (or maybe I misunderstood the manifestation) 

![flyout](https://user-images.githubusercontent.com/471693/121394365-2d048a80-c949-11eb-8609-39f6f32a6c1c.gif)

I've changed the layout of the components regardless though.

Reply:
I probably explained it incorrectly. It happens when you go through the process of creating the ML jobs. The whole page template will re-render a different branch of the large conditional in the page component because the jobs now exist, which causes the fly-out to be re-mounted in a different hierarchy location and trigger the slide-in animation several times.